### ISSUE TITLE
make install : btdht/krcp.pyx:1472:29: Cannot assign None to int

### ISSUE THREAD

**Issue by w888** [nitmir/btdht#8]
hi

i gotten error, when doing make on the latest code, can you help?

```
 Error compiling Cython file:
 ------------------------------------------------------------
 ...
                 else:
                     self.addr_addr_2 = addr[0]
                 self.addr_port = addr[1]
         def __del__(self):
             self.addr_addr = None
             self.addr_port = None
                             ^
 ------------------------------------------------------------

 btdht/krcp.pyx:1472:29: Cannot assign None to int
 Compiling btdht/dht.pyx because it changed.
 Traceback (most recent call last):
   File "setup.py", line 28, in <module>
 Compiling btdht/krcp.pyx because it changed.
 Compiling btdht/utils.pyx because it changed.
 [1/3] Cythonizing btdht/dht.pyx
 [2/3] Cythonizing btdht/krcp.pyx
     ext_modules = cythonize("btdht/*.pyx") if has_cython else c_extensions,
   File "/usr/lib64/python2.7/site-packages/Cython/Build/Dependencies.py", line 1026, in cythonize
     cythonize_one(*args)
   File "/usr/lib64/python2.7/site-packages/Cython/Build/Dependencies.py", line 1146, in cythonize_one
     raise CompileError(None, pyx_file)
 Cython.Compiler.Errors.CompileError: btdht/krcp.pyx
 make: *** [build] Error 1
```

**Comment by w888** [comment]
I resolve it by using earlier Cython

```
Cython==0.21
```

instead of latest

```
Cython==0.28
```

**Comment by jcea** [comment]
Please, publish a new version or document this issue in the build instructions.

I just hit this too, and this bug report and patch are a year old already :-).

**Comment by milahu** [comment]
> patch are a year old already

the patch is merged but [version 0.3.3](https://github.com/nitmir/btdht/commit/df70dd043551d85c2c3585e99e2a2565cc70570e) is untagged

the old [version 0.3.2](https://github.com/nitmir/btdht/releases/tag/v0.3.2) is broken
